api/meetings/views.py

```diff
@@ -64,17 +61,12 @@ class MeetingList(BaseMeetingView, generics.ListAPIView, ListFilterMixin):
 
     # overrides ListFilterMixin
     def get_default_queryset(self):
-        tags = Tag.objects.filter(
-            abstractnode_tagged__is_public=True,
-            abstractnode_tagged__is_deleted=False,
+        conferences = Conference.objects.filter(
+            is_meeting=True,
+            submissions__is_public=True,
+            submissions__is_deleted=False,
         ).annotate(
-            num_nodes=Count(F('abstractnode_tagged')),
-        ).filter(name=OuterRef('endpoint'))
-
-        conferences = Conference.objects.filter(is_meeting=True).annotate(
-            submissions_count=Subquery(
-                tags.values('num_nodes')[:1], output_field=IntegerField(),
-            ),
+            submissions_count=Count(F('submissions')),
```

Comment:
This was the bad query that was causing the discrepancy between MeetingList and MeetingDetail endpoint (List endpoint annotates the queryset with submission count for sorting, while Detail endpoint calculates on the fly).  Since Nodes aren't actually related to Conferences, and the fact that multiple tags on a project can map to the same Conference is making this Django query about impossible (it's easy in SQL, but still not what we need).

Conferences and AbstractNodes have been related via an m2m field which makes annotating each conference in this queryset with the number of submissions super easy.